### DISCUSSION TITLE
Can power splitter be replaced with this cable?

### DISCUSSION THREAD

**Discussion by KimWash** [tiny-pilot/tinypilot#904] [Q&A] [answered]
I think this cable can split power and data properly.
Can this cable prevent power back flow and replace power splitter?

![photo_2022-02-06 23 52 47](https://user-images.githubusercontent.com/47469682/152686699-5b18686b-7e16-4197-b4ed-2b531a9f4861.jpeg)

**Comment by mtlynch** [comment, marked as answer]
Yes, that's equivalent to the functionality in the TinyPilot Power Connector.
